src/lib/components/workflow/download-event-history-modal.svelte

```diff
@@ -65,5 +67,7 @@
         description={translate('events.readable-description')}
       />
     </RadioGroup>
+
+    <WorkflowRawHistoryLink />
```

Comment:
What was the reasoning on putting in the download modal instead of on JSON page? I could be convinced either way, just curious on adding it here when it's not really downloading

Reply:
I have no strong opinion either way though.